meson_options.txt

```diff
@@ -13,17 +13,13 @@ option('mapper-iface',
        description: 'Object mapper interface')
 
 # Inventory manager
-option('inventory-service',
```

Comment:
What about Nicole/Vesnin?

Reply:
PIM also creates the objects with required interfaces. So, it works properly on those platforms too.